code/datums/elements/shuttle_cling.dm

```diff
@@ -0,0 +1,106 @@
+///Instead of a high move force we just get launched away dramatically because we're that hopeless
+#define SUPER_NOT_HOLDING_ON 0
+///We're not holdin on and will get thrown off
+#define NOT_HOLDING_ON 1
+///We're holding on, but will be pulled slowly
+#define CLINGING 2
+///We're holding on really well and aren't suffering from any pull
+#define ALL_GOOD 3
+
+///Component
```

Comment:
This documentation is worse than useless I have no idea waht "shuttle cling" even means

Reply:
I'm so fucking dumb, I left that so I wouldnt forget to add an actual doc comment